### PR TITLE
feat: Cosmos sdk 0.50/CometBFT 0.38 support

### PR DESCRIPTION
**Pull request by clockworkgr**

### CODE REVIEW

**Review by Pantani — APPROVED** [review]
LGMT

File: src/lib/queries/ibc.ts [hunk 1]
```diff
@@ -632,24 +630,26 @@ export function setupIbcExtension(base: QueryClient): IbcExtension {
...
             channelId: string,
-            sequence: Long,
+            sequence: bigint,
             proofHeight: Height
           ) => {
             const key = toAscii(
-              `commitments/ports/${portId}/channels/${channelId}/sequences/${sequence.toNumber()}`
+              `commitments/ports/${portId}/channels/${channelId}/sequences/${Number(
+                sequence
+              )}`
```
Comment (webmaster128) [inline src/lib/queries/ibc.ts:647]: Here you can probably just put `${sequence}`

File: src/lib/ibcclient.ts [hunk 2]
```diff
@@ -1285,7 +1269,7 @@ export class IbcClient {
       const acknowledgement = acks[i].acknowledgement;
 
       this.logger.verbose(
-        `Ack packet #${packet.sequence.toNumber()} from ${this.chainId}:${
+        `Ack packet #${Number(packet.sequence)} from ${this.chainId}:${
```
[the remark this answers is not in the document]
Comment (clockworkgr) [reply src/lib/ibcclient.ts:1272]: You are correct....I thought the `n` would be rendered fot bigints but turns out its not...I'll fix em

**Review by webmaster128 — APPROVED** [review]
Nice, thank you

**Pull request merged**